src/types/IInfographicsDetails.ts

```diff
@@ -0,0 +1,6 @@
+import {TInfographics} from './IInfographicsSection'
+
+export interface IInfographicsDetails {
+    widthPx: number,
```

Comment:
k `width` nemusis pridavat `px`, kdyz to hodis do style tak tam defaultne je value `px`, takze ti staci tam davat jen cislo a pojmenovat to `width`, ale to je samozrejme jen rada, necham na tobe